public/js/clients/chrome.js

```diff
@@ -28,7 +28,8 @@ function presentTabs(tabs) {
         title: tab.title,
         url: tab.url,
         id: tab.id,
-        chrome: tab
+        tab,
+        browser: "chrome"
```

Comment:
I'm not thrilled with having a tab property in tab, but I'm not sure if there is something better. I'm still waiting on what patterns emerge with sources and other state before i make some decisions